tests/test_tpt_reports.py

```diff
@@ -25,6 +25,7 @@
 RELEASE_TAG = os.getenv("RELEASE_TAG")
 PROJECT_VERSION = tpt_reports.__version__
 TEST_JSON_FILE = "tests/data/test.json"
+TEST_BAD_JSON_FILE = "tests/data/bad_file.json"
 DEFAULT_OUTPUT_DIRECTORY = "~/"
```

Comment:
I would alphabetize these now that we have a growing set of them:
```suggestion
DEFAULT_OUTPUT_DIRECTORY = "~/"
PROJECT_VERSION = tpt_reports.__version__
RELEASE_TAG = os.getenv("RELEASE_TAG")
TEST_BAD_JSON_FILE = "tests/data/bad_file.json"
TEST_JSON_FILE = "tests/data/test.json"
```

Reply:
Suggestion applied! Thanks!